src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
```

Comment:
The `Session` object is initialized to `null`. Make sure that the `factory` field in the `AbstractDao` class is properly initialized to avoid `NullPointerException` when calling `factory.openSession()`.